ui/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "experimentalDecorators": true,
     "target": "es2015",
     "typeRoots": ["node_modules/@types"],
-    "types": [],
+    "types": ["cypress", "@testing-library/cypress"],
```

Comment:
Adding these types gave me a warning when running `yarn build`. Can you add them to the `tsconfig.spec.json` instead.

```
Warning: /Users/adandac1/workspace/retroquest-rkennel/ui/node_modules/cypress/types/net-stubbing.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.
```

Broader vision, now that we are not using the angular builders for unit or e2e testing, we should take a look at the tsconfig setting. My assumption is we can simplify them.

Reply:
Would you like to add an issue for that?  Given your recent dive into all things platform, I think you have more expertise on this topic and can provide greater context to the issue.

Reply:
Yeah I will create the issue for cleaning up the tsconfig. For this PR, just move the types to the `tsconfig.spec.json` file.